srv/components/system/files/confstore_template.j2

```diff
@@ -91,6 +91,7 @@ server_node>{{ machine_id }}>network>data>transport_type=>{{ pillar['cluster'][n
 server_node>{{ machine_id }}>storage>cvg_count=>{{ pillar['cluster'][node]['storage']['cvg'] | length }}
 {%- for i in range(pillar['cluster'][node]['storage']['cvg'] | length) %}
 {%- set outer_loop = loop %}
+{%- set cvg_name = pillar['cluster'][node]['storage']['cvg']['i']['cvg_name'] %}
```

Comment:
I think we need to fetch CVG name and set to ConfStore something like:
`server_node>{{ machine_id }}>storage>cvg[{{ outer_loop.index - 1 }}]>name=>{{ pillar['cluster'][node]['storage']['cvg'][outer_loop.index - 1]['cvg_name'] }}`

Reply:
Addressed.